### PR TITLE
Add a filter option to exclude system framework apps.

### LINKED ISSUE #829: [FR] Add filter option in advanced menu
### Checklist / 检查清单

- [X] No one has submitted a similar or identical feature request before. / 之前没有人提交过类似或相同的功能请求。
- [X] This suggestion does not depart from the original intention of LibChecker. / 这个建议不会背离 LibChecker 的初衷。

### Enhancement propose / 改进目的

"Show system framework apps" which package name contains "android"

### Solution / 解决方案

_No response_

### Additional info / 额外信息

_No response_

### PR DESCRIPTION
**Pull request by zhaobozhen**
Closes #829

### CODE REVIEW

File: app/src/main/kotlin/com/absinthe/libchecker/ui/fragment/applist/AppListFragment.kt [hunk 1]
```diff
@@ -366,6 +366,9 @@ class AppListFragment :
     if ((options and AdvancedOptions.SHOW_SYSTEM_APPS) == 0) {
       filterList = filterList.filter { !it.isSystem }.toMutableList()
     }
+    if ((options and AdvancedOptions.SHOW_SYSTEM_FRAMEWORK_APPS) == 0) {
+      filterList = filterList.filter { !it.packageName.contains("android") }.toMutableList()
+    }
```
Comment (qhy040404) [inline app/src/main/kotlin/com/absinthe/libchecker/ui/fragment/applist/AppListFragment.kt]: Some apps contains `android` in their package names, e.g. Apple Music (com.apple.android.music), PayPal (com.paypal.android.p2pmobile)

Comment (zhaobozhen) [reply app/src/main/kotlin/com/absinthe/libchecker/ui/fragment/applist/AppListFragment.kt]: Weird 🤥.
AFAIK, Google Play bans three-party apps with the word "android" in the package name.

Comment (Goooler) [reply app/src/main/kotlin/com/absinthe/libchecker/ui/fragment/applist/AppListFragment.kt]: `it.packageName.startWith("com.android.")` ?

Comment (zhaobozhen) [reply app/src/main/kotlin/com/absinthe/libchecker/ui/fragment/applist/AppListFragment.kt]: Maybe `!it.packageName.startWith("com.android.") && !it.packageName == "android"` is better

Comment (qhy040404) [reply app/src/main/kotlin/com/absinthe/libchecker/ui/fragment/applist/AppListFragment.kt:371]: Then maybe we should rename this filter to `AOSP apps`

**Pull request merged**